src/Title/Title.js

```diff
@@ -2,15 +2,12 @@ import './Title.scss'
 
 const Title = (props) => {
     return (
-        <div>
-            <h1 className={`banner-title align-middle landing-page-title ${props.displayRestaurantsOrMenu}`}>Food. Delivered.</h1>
-            <p className={`align-middle fs-5 mb-0 catch-phrase ${props.displayRestaurantsOrMenu}`}>
-              Order your favourite food from local restaurants, right to your door.
-            </p>
-            <h1 className={`banner-title align-middle title-restaurant-name ${props.displayRestaurantsOrMenu}`}>
-              {props.restaurantDetails?.restaurant ?? ''}
-            </h1>
-        </div>
+        <div>      
+        <h1 className={`banner-title align-middle landing-page-title`}> {props.restaurantDetails?.restaurant ?? "Food. Delivered."}</h1>
+        <p className={`align-middle fs-5 mb-0 catch-phrase ${props.displayRestaurantsOrMenu}`}>
+        {props.restaurantDetails? "": "Order your favourite food from local restaurants, right to your door."}
```

Comment:
you need on both sides of the ? and the : so that this ternary looks properly like a ternary, it's a little confusing at the moment

Reply:
Done

Reply:
done